doc/LanguageClient.txt

```diff
@@ -228,10 +228,11 @@ Valid options: "fzf" | "quickfix" | "location-list" | |Funcref|
 If you use a |Funcref|, the referenced function should have two arguments
 (source, sink). The "source" argument can be either a command string or a
 list. The sink argument can be a command string or a |Funcref| that takes the
-selected line as its first argument.
+selected line as its argument.
```

Comment:
With `sink*` instead of `sink` being passed into `fzf#run(fzf#wrap..`, the selected line is actually the second argument to the sink. The first argument is the `action` users took in the fzf window. The arguments and their order that the `sink` takes should depend on the implementation of the referenced function (`Funcref`).